### PR TITLE
Rewritten virt-sysprep feature page to reflect the current design and…

### PR DESCRIPTION
**Pull request by smelamud**
Changes proposed in this pull request:

-  Rewritten virt-sysprep feature page to reflect the current design and implementation of the feature.

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @smelamud

This pull request needs review by: @maorlipchuk, @laravot

### CODE REVIEW

**Comment by mykaul** [comment]
@JohnMarksRH - can you review?

File: source/develop/release-management/features/virt-sysprep.html.md [hunk 1]
```diff
@@ -1,95 +1,68 @@
...
-<!-- -->
+*   Add `seal` flag to VM Template creation operation.
 
-*   UI desing:
+### Backend
 
-TBD.
+*   Create `VirtJobCallback`. Make `VirtJobCallback` and `StorageJobCallback` to be inherited from common ancestor `HostJobCallback` that will contain the common functionality.
+*   Create `UpdateVolumeCommand` that can change volume flags - to make it ILLEGAL, LEGAL and SHARED.
+*   Create `SealVmTemplateCommand` that invokes VM.seal verb on VDSM side.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/virt-sysprep.html.md]: Create `SealVmTemplateCommand` that invokes VM.seal verb on the VDSM side.

File: source/develop/release-management/features/virt-sysprep.html.md [hunk 2]
```diff
@@ -1,95 +1,68 @@
...
+### VDSM
 
-*   All actions are related to multiple VMs.
-    1.  all of the action of virt-sysprep tab can be run on multiple VMs
-    2.  all the virt-sysprep tab options can be run on a VM-Pool. (ie on all of its VMs)
-    3.  all actions will be enabled for Templates as well
-*   Consider adding support for creating VM from template
-*   VM must be down in order to run virt-sysprep
-*   When adding/update package need to check that the storage domain have free space
+*   Add `SDM.update_volume` verb that changes volume attributes.
```
Comment (JohnMarksRH) [inline source/develop/release-management/features/virt-sysprep.html.md]: Add the `SDM.update_volume` verb that changes volume attributes.

**Comment by michalskrivanek** [comment]
This feature is release, why hold back the documentation?

**Comment by michalskrivanek** [comment]
Who can merge this, why does it take so much time???

**Comment by mykaul** [comment]
@michalskrivanek - several reasons:
1. There were many review comments. Not all of which were attended ASAP.
2. No one from virt team reviewed it.

Right now, it's waiting for CI, then it can be merged.

**Comment by michalskrivanek** [comment]
I hope CI is not running for 2 days....apparently it broke. 
As for review - I still don't think it should be mandatory. People are notified, they can comment, there was enough time for that, but nothing happened. From virt team...well it's written by a virt team" member, the feature owner, so no matter what's the actual content it is ok from "virt team perspective". 
It looks good to me now, but I would support it even before without reading it - it is released and the fact that it is missing is worse than any potentially problematic statements/omissions in the original version. I would prefer an iterative approach of getting it in and then solve details, stylistics, etc. in another PR.

**Comment by mykaul** [comment]
@michalskrivanek - I too, would have preferred seeing this in before the feature was written.
I disagree it should not be reviewed by a peer in the team - prior to language and grammar and look and feel review by others. I agree it could be done iteratively, except for the initial review.

**Pull request merged**